server/events/markdown_renderer.go

```diff
@@ -16,15 +16,26 @@ package events
 import (
 	"bytes"
 	"fmt"
+	"regexp"
 	"strings"
 	"text/template"
 
 	"github.com/Masterminds/sprig"
+	"github.com/runatlantis/atlantis/server/events/models"
 )
 
 const (
 	planCommandTitle  = "Plan"
 	applyCommandTitle = "Apply"
+	// maxUnwrappedLines is the maximum number of lines the Terraform output
+	// can be before we wrap it in an expandable template.
+	maxUnwrappedLines = 12
```

Comment:
Mostly just from looking at it in GitHub. Above 12 lines the comment starts to take more than the whole screen.

Reply:
@lkysow is this configurable via Atlantis yaml per project?

Reply:
No, not right now. What did you want to configure it to?

I'm wary of providing too many configuration knobs but happy to hear your use-case.

Reply:
We have a lot of cases where there are 10+ terraform outputs (successful and failed plans and applies for multiple workspaces) on a single gitlab merge request.

This leads to merge request pages that are _very long_ and a hassle to read through.

I would prefer all terraform output is nested in a collapsible details block. Making this setting configurable as an integer would allow this (for our project I’d specify this as `0` libes to trigger folding).

Reply:
Okay so most of your outputs are less than 12 lines?

Reply:
Can you create an issue for this request please.

Reply:
@lkysow I logged https://github.com/runatlantis/atlantis/issues/321